packages/runtime/src/lib/txn.ts

```diff
@@ -191,3 +193,178 @@ export function isEncTxAssetConfig (txn: EncTx): boolean {
     (txn.caid !== undefined && txn.caid !== 0) && // assetIndex should not be 0
     !isEncTxAssetDeletion(txn); // AND should not be asset deletion
 }
+
+export function isEncTxApplicationCreate (txn: EncTx): boolean {
+  return txn.type === 'appl' && (txn.apan === 0 || txn.apan === undefined);
+}
+
+export function transactionAndSignToExecParams (
+  txAndSign: types.TransactionAndSign, ctx: Context, line?: number
+): types.ExecParams {
```

Comment:
As a good practice let's try to add Description in functions, along with the parameters.